src/Utils/ApiRequests.jsx

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+const ApiRequests = {
+    getArticles() {
+        return axios.get(`https://news-app-msy.herokuapp.com/api/articles`)
```

Comment:
Would be nice to use axios.create here to setup a base url that you can chain requests to - it'll make your code much DRY-er